.github/workflows/lucode2-check.yaml

```diff
@@ -11,7 +11,7 @@ on:
 env:
   USE_BSPM: "true"
   _R_CHECK_FORCE_SUGGESTS_: "false"
-  NO_BINARY_INSTALL_R_PACKAGES: 'c("madrat", "magclass", "citation", "gms", "goxygen")'
+  NO_BINARY_INSTALL_R_PACKAGES: 'c("madrat", "magclass", "citation", "gms", "goxygen", "GDPuc")'
```

Comment:
do you need the GDPuc reference here?

Reply:
no idea how that was added?
It was no change that I did by hand, but rather the outcome of lucode2::buildLibrary() 

if you have an idea how I could remove it, I will be happy to do so.

Reply:
I would ask somebody from the RSE group about that (you can use the mattermost channel). 
Adding strict dependencies is not desirable in any library. If this is a new library requirement demanded by other code change, it should be added in a PR that explain why it is needed.
As your code does not require GDPuc in any way, I would simply remove this change from your PR, and commit only the removal without the need to update any version number.